src/flexbox/nasa.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Nasa</title>
+  <link rel="stylesheet" href="./nasa.styles.css">
+</head>
+<body>
+ 
+  <div class="header">
+    <img src="nasa1.jpg" alt="nasa1">
+  <div class="text">NASA</div>
+  </div>
+
+  <div class="bottom">
+    <div class="h1">
+     <p>SOCIAL MEDIA</p>
+     <p>OFFICE INFO</p>
+     <p>CONTACT US</p>
+    </div>
+    
+    <div class="collum1">
+      <h2>Twitter - @nasa</h2>
```

Comment:
add the title for each section in the collumx divs
`<div class="collum1">
      <h2>SOCIAL MEDIA</h2>
      <h2>Twitter - @nasa</h2>
      <h2>Facebook - fb.com/nasa</h2>
      <h2>Instragram - @nasa</h2>
      <h2>Youtube - youtube.com/NASA</h2>
    </div>`